controllers/gslb_controller_test.go

```diff
@@ -69,6 +69,7 @@ var predefinedConfig = depresolver.Config{
 	ClusterGeoTag:           "us-west-1",
 	ExtClustersGeoTags:      []string{"us-east-1"},
 	EdgeDNSServer:           "8.8.8.8",
```

Comment:
Shouldn't we make `localhost` here? Or use `53` and override `EdgeDNSServerPort` in specific tests?
Otherwise `8.8.8.8` with `7753` is actually never working combination.

Reply:
@ytsarev , thx, amended!

regions:
uk -> za

EdgeDNServer I put there more reasonable domain name.  Tests for localhost EdgeDNSServer are already implemented:  https://github.com/k8gb-io/k8gb/pull/505/files#diff-384d90ba8267cb88a0aab2845e0778c734551e380ce8040b033b006584153fdeR1418